src/redisai.h

```diff
@@ -273,6 +278,7 @@ static int RedisAI_Initialize(RedisModuleCtx *ctx) {
     REDISAI_MODULE_INIT_FUNCTION(ctx, TensorGetValueAsDouble);
     REDISAI_MODULE_INIT_FUNCTION(ctx, TensorGetValueAsLongLong);
     REDISAI_MODULE_INIT_FUNCTION(ctx, TensorGetShallowCopy);
+    REDISAI_MODULE_INIT_FUNCTION(ctx, TensorGetDLTensor);
```

Comment:
why this is on redisai.h?

Reply:
I added this to the LLAPI when I saw that I need it, but I guess that we don't need it there if we have `backends_api.h`...